web/public/locales/en.json

```diff
@@ -91,7 +91,11 @@
     }
   },
   "time-controller": {
-    "title": "Display data from the past"
+    "title": "Display data from the past",
```

Comment:
I think we need to change this title a bit, it kind of clashes with the new wording.

Maybe "Select time resolution" or something along those lines?

Reply:
Good point on changing it. I think "time resolution" is too technical for most people though :)

I'm also wondering if we should do this in a separate PR afterwards instead, as it could potentially be a longer discussion and requires new translations?

Reply:
Hey @electricitymaps/advocacy what do you guys think here?

Should we try and come up with a new title in this PR or should we hold off and do it separately later on?

PS: Trying our new stuff 😅